composer/trainer/dist_strategy.py

```diff
@@ -221,7 +221,7 @@ def prepare_fsdp_module(
     is_torch_2_0 = using_torch_2()
     from torch.distributed.algorithms._checkpoint.checkpoint_wrapper import (CheckpointImpl,
                                                                              apply_activation_checkpointing,
-                                                                             checkpoint_wrapper)
+                                                                             checkpoint_wrapper, offload_wrapper)
```

Comment:
can you verify this fn exists in all torch versions (>=1.13?)

Reply:
it turned out the offload_wrapper was added after 2.0.0, for 1.13, we will use the old implementation. I will add a guard on versions

Reply:
Should import be inlined